introduction.md

```diff
@@ -0,0 +1,2 @@
+I am amerus, and I am a Swiss Army knife.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
https://www.codeschool.com/learn/ruby